### PR TITLE
Add Rainbow-Blocks and Elytras pages

### PR DESCRIPTION
**Pull request by svr333**
## Additions
- Add Rainbow-Blocks.md
- Add Elytras.md

If there is anything I should add on there (pages feel a little empty), feel free to tell me

### CODE REVIEW

File: pages/Elytras.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Obtaining & Usage
+These items can be found in the [Magical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Magical-Gadgets) category. Their exact recipes can be found in-game.  
+The Elytra Scale is a crafting component, while the other three are usable elytras.
+
+| Item | Specialities |
+| ---- | -------- |
+| Elytra | N/A |
+| Infused Elytra | comes with mending |
+| Soulbound Elytra | kept on death |
```
Comment (LinoxGH) [inline pages/Elytras.md:15]: I think these should be capitalized:

```suggestion
| Infused Elytra | Comes with Mending |
| Soulbound Elytra | Kept on Death |
```

File: pages/Rainbow-Blocks.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Rainbow Concrete
+- Rainbow Terracotta
+- Rainbow Wool
+
+## Obtaining
+These items are part of the [Magical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Magical-Gadgets) category. Their exact recipes can be found in-game.  
+Do note that these also exist for specific seasons (eg. Halloween Rainbow blocks). These can be found in their corresponding [Seasonal Categories](https://github.com/Slimefun/Slimefun4/wiki/Seasonal-Categories).
+
+## Usage & Specifications
+These blocks will cycle through all the colours in which they can possibly come.
```
Comment (LinoxGH) [inline pages/Rainbow-Blocks.md:14]: Should be more informative. For example, add the colors and their order.

File: pages/Rainbow-Blocks.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- Rainbow Terracotta
+- Rainbow Wool
+
+## Obtaining
+These items are part of the [Magical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Magical-Gadgets) category. Their exact recipes can be found in-game.  
+Do note that these also exist for specific seasons (eg. Halloween Rainbow blocks). These can be found in their corresponding [Seasonal Categories](https://github.com/Slimefun/Slimefun4/wiki/Seasonal-Categories).
+
+## Usage & Specifications
+These blocks will cycle through all the colours in which they can possibly come.  
+When placed, all blocks of the same type (eg. all Rainbow Wool blocks) will synchronize and display the same colour.
```
Comment (LinoxGH) [inline pages/Rainbow-Blocks.md:15]: Missing <br> in the end.

Comment (svr333) [reply pages/Rainbow-Blocks.md:15]: missing what? @LinoxGH

**Review by LinoxGH — APPROVED** [review]

**Review by Sfiguz7 — APPROVED** [review]

**Pull request merged**